src/event_loop.rs

```diff
@@ -334,6 +334,11 @@ impl<T> EventLoopWindowTarget<T> {
         ))]
         self.p.set_device_event_filter(_filter);
     }
+
+    /// Returns a [`raw_window_handle::RawDisplayHandle`] for the event loop.
+    pub fn raw_display_handle(&self) -> raw_window_handle::RawDisplayHandle {
+        self.p.raw_display_handle()
+    }
```

Comment:
Why did you choose an inherent method instead of using the `HasRawDisplayHandle` trait?

Reply:
I'm still not sure about those, though I think using trait methods is preferable way, I'd bring them back.

Reply:
We "brought them back" for `android-ndk-rs` since nothing in `raw-window-handle` states that these traits are suddenly deprecated (they're for ease of use when passing a window type, though I agree that it's not absolutely necessary since `raw_window_handle::RawDisplayHandle` might just as well be accepted by APIs without having to deal with borrows).

This is public API for the event loop, right? Not sure if every type in `winit` should implement this, for example a single event loop can have multiple windows associated so having the trait/function doesn't make much sense?